### DISCUSSION TITLE
Videos wont download?

### DISCUSSION THREAD

**Discussion by jeezus123** [aclap-dev/video-downloadhelper#430] [Help and Q&A]
### Your question

I've never had this problem before. When i click download, i get this "exit code: 1" error code and it wont download. I have tried to download a video on 2 different computers and it still doesnt download. Ive literally downloaded this video 3 times already this week and i need to download it again because i accidentally deleted the files. So i know its downloadable. What do i do to fix this? please help

this is the error code: 

Error: Exit code: 1

    at as.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624"

![56765756](https://github.com/aclap-dev/video-downloadhelper/assets/168155289/16fa28d2-b5fb-417a-b56e-26c69cd19a33)
![980890098](https://github.com/aclap-dev/video-downloadhelper/assets/168155289/f55efcdd-45e9-4544-81d3-68920383a2eb)
![erro12](https://github.com/aclap-dev/video-downloadhelper/assets/168155289/538f29dc-60b3-468e-9c62-c91e20ca876f)

### Addon & CoApp version and details

I've never had this problem before. When i click download, i get this "exit code: 1" error code and it wont download. I have tried to download a video on 2 different computers and it still doesnt download. Ive literally downloaded this video 3 times already this week and i need to download it again because i accidentally deleted the files. So i know its downloadable. What do i do to fix this? please help

this is the error code: 

Error: Exit code: 1

    at as.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624"

![56765756](https://github.com/aclap-dev/video-downloadhelper/assets/168155289/16fa28d2-b5fb-417a-b56e-26c69cd19a33)
![980890098](https://github.com/aclap-dev/video-downloadhelper/assets/168155289/f55efcdd-45e9-4544-81d3-68920383a2eb)
![erro12](https://github.com/aclap-dev/video-downloadhelper/assets/168155289/538f29dc-60b3-468e-9c62-c91e20ca876f)

**Comment by Mattmjs** [comment]
If you want help then you also need to share link to where the problem is happening. You've also left out your OS and companion version.

**Comment by jeezus123** [comment]
what do you mean a link to where the problem is happening? and what is os
and companion version?

**Reply by Mattmjs** [reply]
I mean, are you on Windows,Mac or Linux ? the companion app version your using ? The link as in the website address where
you are getting the exit code 1 error. 

This is my setup: 
version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
